README.md

```diff
@@ -1,6 +1,14 @@
 # Spandrel
 
-[![PyPI package](https://img.shields.io/badge/pip%20install-spandrel-brightgreen)](https://pypi.org/project/spandrel/) [![version number](https://img.shields.io/pypi/v/spandrel?color=green&label=version)](https://github.com/chaiNNer-org/spandrel/releases) [![Actions Status](https://github.com/chaiNNer-org/spandrel/workflows/Test/badge.svg)](https://github.com/chaiNNer-org/spandrel/actions) [![License](https://img.shields.io/github/license/chaiNNer-org/spandrel)](https://github.com/chaiNNer-org/spandrel/blob/main/LICENSE)
+[![PyPI package](https://img.shields.io/badge/pip%20install-spandrel-brightgreen)](https://pypi.org/project/spandrel/)
+[![version number](https://img.shields.io/pypi/v/spandrel?color=green&label=version)](https://github.com/chaiNNer-org/spandrel/releases)
+![PyPi Downloads](https://img.shields.io/pypi/dw/spandrel)
+![PyPi Wheel](https://img.shields.io/pypi/wheel/spandrel)
+![Python Version](https://img.shields.io/pypi/pyversions/spandrel)
```

Comment:
I don't like that these 3 don't link to anything. When you click on them, you'll just see the image.

I'm also not sure about "wheel | yes". What does that even mean?

Reply:
Some packages release things other than wheels. I agree it isn't that useful, I only did it cuz syrupy did it

Reply:
> Some packages release things other than wheels.

This doesn't really apply to us, so I think we can remove it.

Reply:
Neither of these are really meant to link to anything, they're just visual information. But i'll add links to stuff anyway